docs/serializations.md

```diff
@@ -33,4 +33,28 @@ Canonical serialization is in JSON format, as defined in RFC 8259 (IETF STD 90),
 - integers: represented in base 10 using decimal digits. This designates an integer component that may be prefixed with an optional minus sign. Leading zeros are not allowed.
 - strings: UTF-8 representation without specific canonicalisation. A string begins and ends with quotation marks (%x22). Any Unicode characters may be placed within the quotation marks, except for the two characters that MUST be escaped by a reverse solidus: quotation mark, reverse solidus, and the control characters (U+0000 through U+001F).
 - arrays: An array structure is represented as square brackets surrounding zero or more items. Items are separated by commas.
-- objects: An object structure is represented as a pair of curly brackets surrounding zero or more name/value pairs (or members). A name is a string containing only ASCII characters (0x21-0x7F). The names within an object must be unique. A single colon comes after each name, separating the name from the value. A single comma separates a value from a following name. The name/value pairs are ordered by name.
\ No newline at end of file
+- objects: An object structure is represented as a pair of curly brackets surrounding zero or more name/value pairs (or members). A name is a string containing only ASCII characters (0x21-0x7F). The names within an object must be unique. A single colon comes after each name, separating the name from the value. A single comma separates a value from a following name. The name/value pairs are ordered by name.
+
+## 4.4 Serialization information
+
+A collection of elements may be serialized in multiple formats.
+
+An SpdxDocument element represents the common properties of a collection of
+elements across all serialization data formats within the model.
+
+The actual serialized bytes is represented by an Artifact element within the
+model.
+
+A Relationship of type serializedInArtifact links an SpdxDocument to one or
+more serialized forms of itself.
+
+When serializing a physical SpdxDocument, any property of the logical element
+that can be natively represented within the chosen serialization format
+(e.g., @context prefixes in JSON-LD instead of the namespaceMap) shall utilize
```

Comment:
I fear this "shall" might produce issues in the future.
I propose we only use "may" instead of forcing this, since we do not have a list of where this applies to.

Reply:
Agree. Done.